ci-operator/jobs/openshift/osde2e/openshift-osde2e-main-periodics.yaml

```diff
@@ -3065,6 +3067,67 @@ periodics:
     - name: osde2e-static-byo-vpc
       secret:
         secretName: osde2e-static-byo-vpc
+- agent: kubernetes
+  cluster: build03
+  cron: 0 5 * * 1-5
+  decorate: true
+  extra_refs:
+  - base_ref: main
+    org: openshift
+    repo: osde2e
+  labels:
+    pj-rehearse.openshift.io/can-be-rehearsed: "false"
```

Comment:
We can enable this to true so it will run under the pj-rehearse

Reply:
Ahhh, thanks! I've set that now.